rootston/output.c

```diff
@@ -493,6 +522,11 @@ static void render_output(struct roots_output *output) {
 		wlr_renderer_clear(renderer, clear_color);
 	}
 
+	render_layer(output, output_box, &data,
```

Comment:
We don't want to render layers in fullscreen mode.

Reply:
We want to render some layers in fullscreen mode. But yeah